Packs/IP2LocationIO/Integrations/IP2LocationIO/IP2LocationIO.py

```diff
@@ -0,0 +1,207 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+import urllib3
+from typing import Any, Dict, List
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    def get_ip_geolocation(self, ip: str, api_key: str) -> Dict[str, Any]:
+        """Gets the IP geolocation using the '/' API endpoint
+
+        Args:
+            ip (str): IP address to get the geolocation for.
+
+            api_key (str): IP2Location.io API key.
+
+        Returns:
+            dict: dict containing the IP geolocation as returned from the API
+        """
+
+        return self._http_request(
+            method='GET',
+            url_suffix='/',
```

Comment:
What would happen if the base url is provided with the '/' character at the end?

Reply:
Have modified the code to strip the '/' from the end of the base url. Comments removed too.

Reply:
Great!